arctic/store/_version_store_utils.py

```diff
@@ -28,9 +30,13 @@ def checksum(symbol, doc):
     Checksum the passed in dictionary
     """
     sha = hashlib.sha1()
-    sha.update(symbol)
-    for k in sorted(doc.iterkeys(), reverse=True):
-        sha.update(str(doc[k]))
+    sha.update(symbol.encode('ascii'))
```

Comment:
hashlib takes byte/char arrays - you have to convert the symbol (in P3 unicode str) into ascii to encode now. I can't see why it wouldn't return the same value as all strings in P2.7 have to be ascii.